mifospay/src/main/res/layout/activity_mobile_verification.xml

```diff
@@ -18,7 +18,7 @@
             <android.support.v7.widget.Toolbar
                 xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:app="http://schemas.android.com/apk/res-auto"
-                android:id="@+id/toolbar"
+                android:id="@+id/mobile_verification_toolbar"
```

Comment:
The ids of views should ideally follow  \<what>\<where>\<description> pattern. Hence for this view you should first mention `toolbar` and then the description